### DISCUSSION TITLE
Suggestion: Do something with Postmortems

### DISCUSSION THREAD

**Discussion by mikekasprzak** [ludumdare/ludumdare#2001] [Ideas]
Postmortems are posts, but special. They are written to recall and comment on a Ludum Dare event previously participated in. They are potentially more valuable than other posts, 'cause if they were collected and sorted by ratings, new users could learn from them.

**Comment by mikekasprzak** [comment]
Rise from your grave.

**Comment by mikekasprzak** [comment]
Oops, I mislabelled this. This just says do something with postmortems, because they are special.

**Comment by Cerno-b** [comment]
I recently had the idea to collect post-mortems off-site, now I found this issue :). It would be awesome if we could tag feed posts as post-mortem and have some way of displaying all tagged posts. Same for time lapse videos.

Not sure who would be allowed to change these tags though. The authors probably, but what about the numerous authors who forgot to tag their posts? Should everyone be allowed to set a post-mortem tag? How about removing it? Should there be specific moderators who are allowed to tag post mortems? Should we use a bot?

For now we can find post-mortems using google search "mortem site:ldjam.com", but in the future, a proper search function would be helpful, especially for excluding posts that are already tagged.

**Comment by mikekasprzak** [comment]
User patching is on the TODO list. It's similar to a Wiki, but instead of auto-accepting the changes, the user gets a notification of an edit submitted to them. Then it's up to them (or a mod) to approve the change.

**Comment by Cerno-b** [comment]
So in case the user doesn't care (or misses the notification), then a mod would need to set the tag?
Sounds reasonable. Let me know if you need help tagging past post mortem entries (in case manual input is required).

**Comment by mikekasprzak** [comment]
Rather, the mod would accept the submitted change. That'll be one of the mod-specific features, browsing unaccepted patches. Anyone can propose a patch.
